README.md

```diff
@@ -94,29 +94,30 @@ drastic increase of load due to breaking news.**
 
 1. Create an ssh tunnel to the CODE one-off contributions database:
     1. Clone https://github.com/guardian/contributions-platform
-    2. From the contributions-platform project, Run `./contributions-store/contributions-store-bastion/scripts/open_ssh_tunnel.sh -s CODE` (requires [marauder](https://github.com/guardian/prism/tree/master/marauder))
+    1. From the contributions-platform project, Run `./contributions-store/contributions-store-bastion/scripts/open_ssh_tunnel.sh -s CODE` (requires [marauder](https://github.com/guardian/prism/tree/master/marauder))
+    1. If you need to close your tunnel and didn't make a note of the process number you can run `ps aux | grep amazonaws.com` to find the process number. The output will look something like this `username      1693   0.0  0.0 34153208    952   ??  Ss    4:20pm   0:00.00 ssh -i /private/var/folders/yv/dbtm9psd5ddbjm_lvcjm9zf1vdng_j/T/security_ssm-scala_temporary-rsa-private-key.tmp -f machine.eu-west-1.compute.amazonaws.com -L 5432:contributions-store-code.address.rds.amazonaws.com:5432 -N -o IdentitiesOnly yes -o ExitOnForwardFailure yes` where `1693` is the process number. You can close the tunnel with `kill 1693`.
 
-1. Ensure an `nginx` service is running locally.
+1. Ensure an `nginx` service is running locally. You can run `dev-nginx restart-nginx` to do this.
 
+### Identity frontend local sign in
+As the /me endpoints use the GU_U and SC_GU_U from the Cookie request header you will need to sign in to the identity-frontend locally.
+1. Start up a local Identity service by running script `start-frontend.sh` in the `identity-frontend` repo.
```

Comment:
Perhaps link to repo to make it one click quicker (unless it's already linked to elsewhere)

Reply:
```suggestion
1. Start up a local Identity service by running script `start-frontend.sh` in the [identity-frontend](https://github.com/guardian/identity-frontend) repo.
```
good spot, thanks!